src/applications/appeals/996/content/areaOfDisagreement.jsx

```diff
@@ -110,3 +109,7 @@ export const AreaOfDisagreementReviewField = ({ children }) => {
     </div>
   ) : null;
 };
+
+AreaOfDisagreementReviewField.propTypes = {
+  children: PropTypes.node.isRequired,
+};
```

Comment:
ooh interesting — was this a linter suggestion or did you just notice that this was missing/needed?

Reply:
hmmm okay 🤔  i wonder if it shouldn't have `isRequired` chained onto it bc on line 102, i'm seeing `children?.props` which kind of suggests that `children` might not be there. thoughts?

Reply:
good catch! I didn't consider the `?` and how it could mean it may not always exist!